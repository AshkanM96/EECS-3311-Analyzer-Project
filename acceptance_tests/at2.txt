divide 
reset
divide
integer_constant (5)
integer_constant (0)
type_check
evaluate
reset
