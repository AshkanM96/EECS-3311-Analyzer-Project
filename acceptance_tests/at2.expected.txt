  Expression currently specified: ?
  Report: Expression is initialized.
->divide
  Expression currently specified: (? / nil)
  Report: OK.
->reset
  Expression currently specified: ?
  Report: OK.
->divide
  Expression currently specified: (? / nil)
  Report: OK.
->integer_constant(5)
  Expression currently specified: (5 / ?)
  Report: OK.
->integer_constant(0)
  Expression currently specified: (5 / 0)
  Report: OK.
->type_check
  Expression currently specified: (5 / 0)
  Report: (5 / 0) is type-correct.
->evaluate
  Expression currently specified: (5 / 0)
  Report: Error (Divisor is zero).
->reset
  Expression currently specified: ?
  Report: OK.
